src/main/java/uk/gov/hmcts/darts/event/service/handler/SentencingRemarksAndRetentionPolicyHandler.java

```diff
@@ -57,12 +57,22 @@ public void handle(final DartsEvent dartsEvent, EventHandlerEntity eventHandler)
         transcriptionRequestDetails.setTranscriptionTypeId(SENTENCING_REMARKS.getId());
         transcriptionRequestDetails.setTranscriptionUrgencyId(STANDARD.getId());
 
-        var transcriptionResponse = transcriptionsApi.saveTranscriptionRequest(transcriptionRequestDetails, false);
+        try {
+            var transcriptionResponse = transcriptionsApi.saveTranscriptionRequest(transcriptionRequestDetails, false);
+            var updateTranscription = new UpdateTranscriptionRequest();
+            updateTranscription.setTranscriptionStatusId(APPROVED.getId());
+            updateTranscription.setWorkflowComment("Transcription Automatically approved");
+            transcriptionsApi.updateTranscription(transcriptionResponse.getTranscriptionId(), updateTranscription);
 
-        var updateTranscription = new UpdateTranscriptionRequest();
-        updateTranscription.setTranscriptionStatusId(APPROVED.getId());
-        updateTranscription.setWorkflowComment("Transcription Automatically approved");
-        transcriptionsApi.updateTranscription(transcriptionResponse.getTranscriptionId(), updateTranscription);
+        } catch (DartsApiException dartsException) {
```

Comment:
I wonder whether it would be better to change the `TranscriptionService.saveTranscriptionRequest` itself to just "do nothing" in case of duplicates, if that is not the new business rule. 
The `DuplicateRequestDetector` rather than throwing an exception (which will be always ignored now), can return a boolean used by the service to skip the save. 
This would probably lead to less code, and less duplication (having to catch/ignore/re-throw TranscriptionApiError.DUPLICATE_TRANSCRIPTION everywhere we use the service). It also avoids potential bugs in the future in case a new client of the service forgets to catch/ignore/re-throw